### PR TITLE
0.10 into develop

### PR DESCRIPTION
**Pull request by rtibbles**
# Conflicts:
#	kolibri/__init__.py
#	kolibri/core/assets/src/styles/main.styl
#	kolibri/logger/signals.py
#	requirements/base.txt

### CODE REVIEW

**Review by indirectlylit — APPROVED** [review]
thanks!

File: kolibri/core/assets/test/progress-icon.spec.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import ProgressIcon from '../src/views/progress-icon';
 
 function testIcon(wrapper, expectedText) {
   // prettier-ignore
-  expect(wrapper.find(UiTooltip).text().trim()).to.equal(expectedText);
+  expect(wrapper.find(UiTooltip).text().trim()).toEqual(expectedText);
```
Comment (rtibbles) [inline kolibri/core/assets/test/progress-icon.spec.js]: :+1:

**Comment by benjaoming** [comment]
Something is wrong with Travis' git data:

image

This is not consistent with what I find when pulling your remote branch @rtibbles 

I'm not sure about the cause or how to fix it. Possibly, try to create a new remote branch and a new PR?

**Comment by rtibbles** [comment]
Closing to do a new merge and sort out this noise.

**Comment by benjaoming** [comment]
@rtibbles seems the later commits passed tests!

**Pull request closed without merging**